src/styles/_catalog.scss

```diff
@@ -1,22 +1,37 @@
+$cardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+
+  justify-content: center;
+
+  @media (max-width: 1023.5px) {
```

Comment:
I think it's better to use numbers like 1024, but not 1023.5, but I'm not really sure about that